community/bolt/src/test/java/org/neo4j/bolt/v1/transport/integration/AuthenticationIT.java

```diff
@@ -115,7 +116,7 @@ public void shouldFailIfWrongCredentials() throws Throwable
         // Then
         assertThat( client, eventuallyRecieves( new byte[]{0, 0, 0, 1} ) );
         assertThat( client, eventuallyRecieves( msgFailure( Status.Security.AuthenticationFailed,
-                "The client provided an incorrect username and/or password." ) ) );
+                String.format( "The client provided an incorrect username and/or password. (ID:%s)", server.uniqueIdentier()) ) ) );
```

Comment:
This is perfect for what we need IMO, easy to parse out and human readable without committing ourselves to an API we don't want to support in the long term.